ext/RMagick/rmimage.c

```diff
@@ -9913,12 +9913,12 @@ Image_page_eq(VALUE self, VALUE rect)
  *
  * Ruby usage:
  *   - @verbatim Image#paint_transparent(target) @endverbatim
- *   - @verbatim Image#paint_transparent(target, opacity) @endverbatim
- *   - @verbatim Image#paint_transparent(target, opacity, invert) @endverbatim
- *   - @verbatim Image#paint_transparent(target, opacity, invert, fuzz) @endverbatim
+ *   - @verbatim Image#paint_transparent(target, alpha) @endverbatim
```

Comment:
@dlemstra should these doc comments show `alpha` as a keyword argument?

```rb
Image#paint_transparent(target, alpha: alpha)
```

Reply:
They probably should. Is that the correct syntax for keyword arguments in docs?

Reply:
I'm not familiar with this documentation syntax. Looks like it's [Doxygen](https://github.com/analogdevicesinc/Linduino/blob/master/LTSketchbook/Documentation/Doxygen/Include/DoxygenReference.md). But regardless, this looks like it's supposed to be possible method signatures, so I'm guessing we probably want to show it as a keyword.